smelter-core/src/pipeline/rtp/rtp_input/jitter_buffer.rs

```diff
@@ -0,0 +1,170 @@
+use std::{
+    collections::BTreeMap,
+    sync::Arc,
+    time::{Duration, Instant},
+};
+
+use tracing::{debug, trace};
+
+use crate::pipeline::{
+    rtp::{
+        RtpPacket,
+        rtp_input::rtcp_sync::{RtpNtpSyncPoint, RtpTimestampSync},
+    },
+    utils::input_buffer::InputBuffer,
+};
+
+use crate::prelude::*;
+
+struct JitterBufferPacket {
+    packet: webrtc::rtp::packet::Packet,
+    pts: Duration,
+    received_at: Instant,
+}
+
+#[derive(Debug, Clone)]
+pub(crate) struct RtpJitterBufferInitOptions {
+    mode: RtpJitterBufferMode,
+    buffer: InputBuffer,
```

Comment:
not really a buffer, it holds information about timestamps/size of the buffer, but it does not do any actuall buffering.

InputBuffer element should be created once for all the tracks(and shared)